File: benchmark/results/java-multilang/run-005/a-mcp/output/shipping/aggregates/Shipment.java
package com.metaengine.demo.shipping.aggregates;

import java.time.OffsetDateTime;

// Shipment aggregate root for the shipping domain.

public class Shipment {
    public String id;
    public OffsetDateTime createdAt;
    public OffsetDateTime updatedAt;
    public String name;
    public String description;

    public Shipment(String id, OffsetDateTime createdAt, OffsetDateTime updatedAt, String name, String description) {
        this.id = id;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
        this.name = name;
        this.description = description;
    }
}

File: benchmark/results/python-invocation-script/run-003/a-mcp/output/shipping/aggregates/shipment.py
from datetime import datetime

"""Shipment aggregate root for the shipping domain."""
class Shipment:
    def __init__(self, id: str, created_at: datetime, updated_at: datetime, name: str, description: str):
        self.id = id
        self.created_at = created_at
        self.updated_at = updated_at
        self.name = name
        self.description = description

